website/blog/2022-06-21-version-069.md

```diff
@@ -0,0 +1,79 @@
+
+## New Architecture
+
+We are continuing the roll-out of the New Architecture for both Android and iOS. If you have not migrated your app or library, yet, follow the steps [here](/docs/new-architecture-intro). You can also read the [latest update](/blog/2022/06/16/resources-migrating-your-react-native-library-to-the-new-architecture) on tools and resources for the New Architecture to learn more.
+
+## Highlights of 0.69
+
+As mentioned above, the most important improvements in this release are centered around React 18 support and bundled Hermes. However, there have been other notable changes, including:
+
+- [Deprecating support](https://github.com/facebook/react-native/commit/5f2835b14d35681c268dd64d6ec284ea5f053be3) for iOS/tvOS SDK 11.0, 12.4+
+- [Better support](https://github.com/facebook/react-native/commit/c5babd993a2bed2994ecc4710fa9e424b3e6cfc2) for M1 users developing for Android
+- [Addition](https://github.com/facebook/react-native/commit/0480f56c5b5478b6ebe5ad88e347cad2810bfb17) of the new `.xcode.env` configuration file for more deterministically sourcing the node executable
+- [React Native now uses](https://github.com/facebook/react-native/commit/50c8e973f067d4ef1fc3c2eddd360a0709828968) the latest status bar API from Android 11
+- [Support for C++17](https://github.com/facebook/react-native/commit/c2e4ae39b8a5c6534a3fa4dae4130166eda15169)
+- [New](https://github.com/facebook/react-native/commit/1a1a304ed2023d60547aef65b1a7bf56467edf08) `hotkeysEnabled` option in the iOS' debug menu
+
+### Breaking changes
+
+There have also been a few breaking changes:
+
+- Support for `console.disableYellowBox` [has been dropped](https://github.com/facebook/react-native/commit/b633cc130533f0731b2577123282c4530e4f0abe)
+- Already deprecated prop types have been removed ([cdfddb4dad](https://github.com/facebook/react-native/commit/cdfddb4dad7c69904850d7e5f089a32a1d3445d1), [3e229f27bc](https://github.com/facebook/react-native/commit/3e229f27bc9c7556876ff776abf70147289d544b), [10199b1581](https://github.com/facebook/react-native/commit/10199b158138b8645550b5579df87e654213fe42))
+- `removeListener`, deprecated since RN0.65, [was removed](https://github.com/facebook/react-native/commit/8dfbed786b40082a7a222e00dc0a621c0695697d) from Appearance
+- If you were using `SegmentedComponentIOS`, you will now need to move to the [segmented-control](https://github.com/react-native-segmented-control/segmented-control) library ([235f168574](https://github.com/facebook/react-native/commit/235f1685748442553e53f8ec6d904bc0314a8ae6))
+
+### Upgrades
+
+And we upgraded some of our dependencies:
+
+- React Native CLI has [been bumped](https://github.com/facebook/react-native/commit/0605880c9ed0aec812f3198eb5075db64fba969a) to [v8.0.0](https://github.com/react-native-community/cli/releases/tag/v8.0.0)
```

Comment:
Yup, it's a good idea to mention breaking changes from the CLI explicitly here 👍🏼

Reply:
Yes yes - this is going to affect a lot of people. If the suggested external module https://github.com/unimonkiez/react-native-asset gets some success reports, it would be even better to point directly to it in order to just pre-answer all the support questions / issues before people even have a chance to log them...

Reply:
```suggestion
```
Since now this is in the breaking changes, I'd remove it from this section